### PR TITLE
Rn722

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by Overtorment** [comment]
A lot of tests failed

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/exzwmuctwhuhepp3mjmosjhmty?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/121)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/es4of4drfamwrex5utxl3vht2a?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/123)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/egv2t23rfq26yg4xzbso4n3r3q?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/124)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/ryka76t4d6qrf5i2cbxsnp4axe?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/125)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/nyxnti2czxznrssgzalvz67fum?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/171)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/nnw75rpsdpoavchdckm27u3xiy?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/173)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/foyyme4eggoxdga4fxohdbcs5u?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/174)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/ettp62kj5gonmguzyy2lhpql5y?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/177)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/ktytw5eess2u3rroijrnpklwj4?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/180)

**Comment by Overtorment** [comment]
@marcosrdz conflicts! also, whats the status on this? ready for review/merge? needs my help on e2e or smth?

**Comment by marcosrdz** [comment]
> @marcosrdz conflicts! also, whats the status on this? ready for review/merge? needs my help on e2e or smth?

Detox lacks rn 722 support for android atm

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/xsjmfv6cpo6etj3n6zltealpmi?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/246)

File: .gitignore [hunk 1]
```diff
@@ -33,6 +33,9 @@ build/
 local.properties
 *.iml
 
+# testing
+/coverage
```
Comment (Overtorment) [inline .gitignore:37]: we dont have coverage, just sayin

File: blue_modules/react-native-bw-file-access/ios/BwFileAccess.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -157,7 +157,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 13.4;
```
Comment (Overtorment) [inline blue_modules/react-native-bw-file-access/ios/BwFileAccess.xcodeproj/project.pbxproj:160]: is  that min os requirement now..?

**Review by Overtorment — APPROVED** [review]
LFG

File: blue_modules/react-native-bw-file-access/ios/BwFileAccess.xcodeproj/project.pbxproj (hunk 2 above)
Comment (marcosrdz) [reply blue_modules/react-native-bw-file-access/ios/BwFileAccess.xcodeproj/project.pbxproj:160]: Yup. Anything lower fails

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

File: .gitignore (hunk 1 above)
Comment (marcosrdz) [reply .gitignore:37]: I just follow what the guide says. Aka "i just work here"

**Pull request merged**